### ISSUE TITLE
Revisit README word cloud

### ISSUE THREAD

**Issue by apulsipher** [EpiForeSITE/software#21]
The word cloud in the README is jumbled and hard to parse. It would be more useful if each project had one or more keywords associated with it and the word cloud was generated from those. 

This issue is to resolve this problem either by using a visualization other than a word cloud or generating it with keywords.

**Comment by JakeWags** [comment]
Came across this and I must say, I hate word clouds 😅 
They serve a very very very specific purpose for visualization, but are mostly for flashy corporate presentations (in my opinion).
Surely there's a better alternative visualization to see keyword makeup in the projects...
This is hardly useful (sorry for my strong criticism, I feel strongly about word clouds)

<img width="1660" height="1660" alt="Image" src="https://github.com/user-attachments/assets/c3741398-2e52-4011-adf2-80e2c2220d06" />
